package.json

```diff
@@ -62,7 +62,7 @@
     "test": "lerna run test --concurrency 1",
     "update-data-packages": "./scripts/update-data-dependencies.sh && yarn && yarn dedupe",
     "update-test-timing-data": "ts-node ./scripts/cci-get-job-metrics.ts && ts-node ./scripts/cci-extract-test-timings-from-job-metrics.ts",
-    "update-versions": "lerna version --yes --no-commit-hooks --no-push --exact --conventional-commits --no-git-tag-version",
+    "update-versions": "lerna version --yes --no-commit-hooks --no-push --exact --conventional-commits --no-git-tag-version --force-publish",
```

Comment:
The command did not work with last flag.

Before change:
```
bcd0740fa72c:amplify-cli-nofork sobkamil$ yarn update-versions
lerna notice cli v6.6.1
lerna info versioning independent
lerna notice Current HEAD is already released, skipping change detection.
lerna success No changed packages to version
bcd0740fa72c:amplify-cli-nofork sobkamil$ yarn lerna version --yes --no-commit-hooks --no-push --exact --no-git-tag-version
lerna notice cli v6.6.1
lerna info versioning independent
lerna notice FYI git repository validation has been skipped, please ensure your version bumps are correct
lerna notice Current HEAD is already released, skipping change detection.
lerna success No changed packages to version
```

After:
```
bcd0740fa72c:amplify-cli-nofork sobkamil$ yarn update-versions
lerna notice cli v6.6.1
lerna info versioning independent
lerna WARN force-publish all packages
lerna info Assuming all packages changed
lerna WARN version Skipping working tree validation, proceed at your own risk
lerna info getChangelogConfig Successfully resolved preset "conventional-changelog-angular"

Changes:
 - @aws-amplify/amplify-app: 5.0.23 => 5.0.24
 - @aws-amplify/amplify-appsync-simulator: 2.15.3 => 2.15.4
 - @aws-amplify/amplify-category-analytics: 5.0.23 => 5.0.24
 - @aws-amplify/amplify-category-auth: 3.7.2 => 3.7.3
 - @aws-amplify/amplify-category-custom: 3.1.11 => 3.1.12
 - @aws-amplify/amplify-category-function: 5.6.2 => 5.6.3
 ...
 ...
```

Reply:
This will likely cause some unnecessary patch version bumps, but that's probably okay

Reply:
That's correct. Lesser evil.